src/components/TitleTransferPanel/Address/Address.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from "react";
+import css from "./Address.module.scss";
+
+interface AddressProps {
+  title: string;
+  name?: string;
+  children: ReactElement;
+}
+
+export const Address = ({ title, name, children }: AddressProps) => {
```

Comment:
Thinking if we want to call this AddressWithContext or something to suggest that it has a name and title?

Reply:
added prettierignore on mdx files instead. ok i rename to AddressInfo